### ISSUE TITLE
Archive "[path]" is invalid after downloading Access db

### ISSUE THREAD

**Issue by CarlosHartmann** [sfa-siard/SiardGui#20]
Using the latest version 2.1.134, Java Version 1.8.0_151, Windows 10 64 bit

I tried downloading a Microsoft Access DB that works when opened alone. There are errors when downloading it. Since the content is not confidential, I uploaded it [here ](https://we.tl/t-TNTDCQWzCr)(please download it within the week before the link expires, the issue itself is not pressing, however).

Downloading it with the "archive views as tables" checkmark produces one error visible in the "Err" tab:

```
Herunterladen der Datenbank gescheitert (  java.sql.SQLException: SQL parsing error at position 2/81 at [@43,374:374='.',<562>,2:81]: mismatched input '.' expecting {<EOF>, AND, CROSS, FULL, GROUP, HAVING, INNER, IS, LEFT, NATURAL, OR, RIGHT, UNION, WHERE, WINDOW, ','}
Rule stack: [sqlStatement, querySpecification, fromClause, tableReference, joinSpecification, booleanValueExpression])! Unvollständiges SIARD-Archiv wird gelöscht.
```

Downloading it without extra options produces the following errors in the "Err" tab, after which it says "erfolgreich heruntergeladen" but shows the error message quoted in the title:

```
View "dbuser"."ArchiveSeries Without Matching Query_branches" could not be archived (  java.sql.SQLException: MS Access query ArchiveSeries Without Matching Query_branches is too complex:   java.lang.IllegalArgumentException: Column type for QUERY_BRANCHES.BRANCH could not be established!)!
View "dbuser"."Find duplicates for View_IT-S_Archivschränke" could not be archived (  java.sql.SQLException: MS Access query Find duplicates for View_IT-S_Archivschränke is too complex:   java.lang.IllegalArgumentException: Column type for REPORTVIEW_ITS_ARCHIVSCHRÄNKE.TECHARCHIVEID could not be established!)!
View "dbuser"."Find duplicates for View_ITS_build" could not be archived (  java.sql.SQLException: MS Access query Find duplicates for View_ITS_build is too complex:   java.lang.IllegalArgumentException: Column type for VIEW_ITS_BUILD.TECHARCHIVEID could not be established!)!
View "dbuser"."List_ForeignPatents" could not be archived (  java.sql.SQLException: MS Access query List_ForeignPatents is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/215 at [@26,215:215='!',<588>,1:215]: mismatched input '!' expecting {AS, FROM, DEREFERENCE_OPERATOR, ',', '.', '['}
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist, valueExpression, commonValueExpression, valueExpressionPrimary, generalValueSpecification, reference, identifierChain])!
View "dbuser"."List_Parent_Children" could not be archived (  java.sql.SQLException: MS Access query List_Parent_Children is too complex:   java.lang.IllegalArgumentException: Column type for QLG_LISTCHILDREN.SERIESID could not be established!)!
View "dbuser"."QA_Keywords_Count" could not be archived (  java.sql.SQLException: MS Access query QA_Keywords_Count is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 2/71 at [@47,235:243='", 0 AS "',<343>,2:71]: mismatched input '", 0 AS "' expecting {<EOF>, CROSS, FULL, GROUP, HAVING, INNER, LEFT, NATURAL, RIGHT, UNION, WHERE, WINDOW, '(', ','}
Rule stack: [sqlStatement, querySpecification, fromClause, tableReference, tablePrimary, tableAlias])!
View "dbuser"."QA_Model_Count" could not be archived (  java.sql.SQLException: MS Access query QA_Model_Count is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 2/75 at [@47,237:245='", 0 AS "',<343>,2:75]: mismatched input '", 0 AS "' expecting {<EOF>, CROSS, FULL, GROUP, HAVING, INNER, LEFT, NATURAL, RIGHT, UNION, WHERE, WINDOW, '(', ','}
Rule stack: [sqlStatement, querySpecification, fromClause, tableReference, tablePrimary, tableAlias])!
View "dbuser"."QA_ParentChild" could not be archived (  java.sql.SQLException: MS Access query QA_ParentChild is too complex:   java.lang.IllegalArgumentException: Column type for QLG_LISTPARENTS.TECHARCHIVEID could not be established!)!
View "dbuser"."QA_ParentChildDetailsWithoutMatchingQuery3" could not be archived (  java.sql.SQLException: MS Access query QA_ParentChildDetailsWithoutMatchingQuery3 is too complex:   java.lang.IllegalArgumentException: Column type for QA_PARENTCHILDDETAILSARCHIVE.PARENT could not be established!)!
View "dbuser"."QD_DMS_SeriesDistribution" could not be archived (  java.sql.SQLException: MS Access query QD_DMS_SeriesDistribution is too complex:   java.lang.IllegalArgumentException: Column type for "View_IT-S_Archivschränke".DMSID could not be established!)!
View "dbuser"."QD_LocationDistribution" could not be archived (  java.sql.SQLException: MS Access query QD_LocationDistribution is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 2/5 at [@33,154:154='"',<552>,2:5]: no viable alternative at input '"'
Rule stack: [sqlStatement, querySpecification, fromClause, tableReference])!
View "dbuser"."QD_PhotoKeywordDistribution" could not be archived (  java.sql.SQLException: MS Access query QD_PhotoKeywordDistribution is too complex:   java.lang.IllegalArgumentException: Cannot evaluate set function COUNT(KEYWORD2PHOTO.RELATIONID)!)!
View "dbuser"."QD_SeriesDistribution" could not be archived (  java.sql.SQLException: MS Access query QD_SeriesDistribution is too complex:   java.lang.IllegalArgumentException: Column type for QL_LISTEXTERNALNODES.HIERARCHY could not be established!)!
View "dbuser"."QD_SeriesDistribution_ExternalNodes" could not be archived (  java.sql.SQLException: MS Access query QD_SeriesDistribution_ExternalNodes is too complex:   java.lang.IllegalArgumentException: Column type for QD_SERIESDISTRIBUTION.SERIESID could not be established!)!
View "dbuser"."QD_SeriesDistribution_InternalNodes" could not be archived (  java.sql.SQLException: MS Access query QD_SeriesDistribution_InternalNodes is too complex:   java.lang.IllegalArgumentException: Column type for QD_SERIESDISTRIBUTION.SERIESID could not be established!)!
View "dbuser"."QLG_List_ParentChildren Without Matching QA_ParentChild" could not be archived (  java.sql.SQLException: MS Access query QLG_List_ParentChildren Without Matching QA_ParentChild is too complex:   java.lang.IllegalArgumentException: Column type for QLG_LIST_PARENTCHILDREN.CHILDID could not be established!)!
View "dbuser"."QL_Bibliothek" could not be archived (  java.sql.SQLException: MS Access query QL_Bibliothek is too complex:   java.lang.IllegalArgumentException: Column type for REPORTVIEW_ITS_BIBLIOTHEK.PUBLISHED could not be established!)!
View "dbuser"."QL_ListArchivalienArt" could not be archived (  java.sql.SQLException: MS Access query QL_ListArchivalienArt is too complex:   java.lang.IllegalArgumentException: Type of ValueExpressionPrimary not supported for evaluation!)!
View "dbuser"."QL_ListArchiveLocations" could not be archived (  java.sql.SQLException: MS Access query QL_ListArchiveLocations is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/56 at [@8,56:56='!',<588>,1:56]: no viable alternative at input 'ArchiveLocation.Room||': '||"ArchiveLocation"!'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist, valueExpression])!
View "dbuser"."QL_ListLanguages" could not be archived (  java.sql.SQLException: MS Access query QL_ListLanguages is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/41 at [@7,41:48='Language',<168>,1:41]: no viable alternative at input 'Languages.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."QLength_Bemerkung" could not be archived (  java.sql.SQLException: MS Access query QLength_Bemerkung is too complex:   java.lang.IllegalArgumentException: Type of ValueExpressionPrimary not supported for evaluation!)!
View "dbuser"."QLength_Beschreibung" could not be archived (  java.sql.SQLException: MS Access query QLength_Beschreibung is too complex:   java.lang.IllegalArgumentException: Type of ValueExpressionPrimary not supported for evaluation!)!
View "dbuser"."QM_ArchiveSeries Without TechnicalArchiveObject" could not be archived (  java.sql.SQLException: MS Access query QM_ArchiveSeries Without TechnicalArchiveObject is too complex:   java.lang.IllegalArgumentException: Column type for QL_LISTEXTERNALNODES.HIERARCHY could not be established!)!
View "dbuser"."QM_Keywords_not_used" could not be archived (  java.sql.SQLException: MS Access query QM_Keywords_not_used is too complex:   java.lang.IllegalArgumentException: Column type for QM_KEYWORDS_NOT_IN_ARCHIVE.KEYWORDID could not be established!)!
View "dbuser"."QQ_MemoLength" could not be archived (  java.sql.SQLException: MS Access query QQ_MemoLength is too complex:   java.lang.IllegalArgumentException: Type of ValueExpressionPrimary not supported for evaluation!)!
View "dbuser"."Q_100_children" could not be archived (  java.sql.SQLException: MS Access query Q_100_children is too complex:   java.lang.IllegalArgumentException: Type of ValueExpressionPrimary not supported for evaluation!)!
View "dbuser"."Q_ArchArt_Distinct" could not be archived (  java.sql.SQLException: MS Access query Q_ArchArt_Distinct is too complex:   java.lang.IllegalArgumentException: Type of ValueExpressionPrimary not supported for evaluation!)!
View "dbuser"."Q_KeywordCount" could not be archived (  java.sql.SQLException: MS Access query Q_KeywordCount is too complex:   java.lang.IllegalArgumentException: Cannot evaluate set function COUNT(KEYWORD2ARCHIVE.KEYWORDID)!)!
View "dbuser"."Q_KeywordsUsed" could not be archived (  java.sql.SQLException: MS Access query Q_KeywordsUsed is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 2/104 at [@28,233:241='", 0 AS "',<343>,2:104]: mismatched input '", 0 AS "' expecting {<EOF>, CROSS, FULL, GROUP, HAVING, INNER, LEFT, NATURAL, RIGHT, UNION, WHERE, WINDOW, '(', ','}
Rule stack: [sqlStatement, querySpecification, fromClause, tableReference, tablePrimary, tableAlias])!
View "dbuser"."Q_LanguageCombinations Without Matching Languages" could not be archived (  java.sql.SQLException: MS Access query Q_LanguageCombinations Without Matching Languages is too complex:   java.lang.IllegalArgumentException: Column type for Q_LANGUAGECOMBINATIONS.SPRACHECODE could not be established!)!
View "dbuser"."Q_LengthArchArt" could not be archived (  java.sql.SQLException: MS Access query Q_LengthArchArt is too complex:   java.lang.IllegalArgumentException: Type of ValueExpressionPrimary not supported for evaluation!)!
View "dbuser"."Q_Loc_LengthArchArt_Distinct" could not be archived (  java.sql.SQLException: MS Access query Q_Loc_LengthArchArt_Distinct is too complex:   java.lang.IllegalArgumentException: Type of ValueExpressionPrimary not supported for evaluation!)!
View "dbuser"."Q_Location_Distr" could not be archived (  java.sql.SQLException: MS Access query Q_Location_Distr is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 2/36 at [@37,193:396='".ParentID, ArchiveLocation.LocationID, ArchiveLocation.Location, ArchiveLocation.SortCode, ArchiveLocation.SortOrder FROM ArchiveLocation INNER JOIN TechnicalArchiveObject ON ArchiveLocation.LocationID="',<343>,2:36]: mismatched input '".ParentID, ArchiveLocation.LocationID, ArchiveLocation.Location, ArchiveLocation.SortCode, ArchiveLocation.SortOrder FROM ArchiveLocation INNER JOIN TechnicalArchiveObject ON ArchiveLocation.LocationID="' expecting {<EOF>, CROSS, FULL, GROUP, HAVING, INNER, LEFT, NATURAL, RIGHT, UNION, WHERE, WINDOW, '(', ','}
Rule stack: [sqlStatement, querySpecification, fromClause, tableReference, tablePrimary, tableAlias])!
View "dbuser"."Q_MixedFamily2" could not be archived (  java.sql.SQLException: MS Access query Q_MixedFamily2 is too complex:   java.lang.IllegalArgumentException: Column type for Q_MIXEDFAMILY.PARENTID could not be established!)!
View "dbuser"."Q_ParentType" could not be archived (  java.sql.SQLException: MS Access query Q_ParentType is too complex:   java.lang.IllegalArgumentException: Cannot evaluate set function COUNT(TECHNICALARCHIVEOBJECT.TECHARCHIVEID)!)!
View "dbuser"."Q_SpracheInHaengereg" could not be archived (  java.sql.SQLException: MS Access query Q_SpracheInHaengereg is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/116 at [@15,116:123='Language',<168>,1:116]: no viable alternative at input 'Languages.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."Q_SpracheInSchraenke" could not be archived (  java.sql.SQLException: MS Access query Q_SpracheInSchraenke is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/116 at [@15,116:123='Language',<168>,1:116]: no viable alternative at input 'Languages.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."Q_SpracheInTresor" could not be archived (  java.sql.SQLException: MS Access query Q_SpracheInTresor is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/116 at [@15,116:123='Language',<168>,1:116]: no viable alternative at input 'Languages.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."Q_SpracheInZeichnung" could not be archived (  java.sql.SQLException: MS Access query Q_SpracheInZeichnung is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/116 at [@15,116:123='Language',<168>,1:116]: no viable alternative at input 'Languages.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."Q_StringlengthArchive" could not be archived (  java.sql.SQLException: MS Access query Q_StringlengthArchive is too complex:   java.lang.IllegalArgumentException: Type of ValueExpressionPrimary not supported for evaluation!)!
View "dbuser"."Q_StringlengthLibrary" could not be archived (  java.sql.SQLException: MS Access query Q_StringlengthLibrary is too complex:   java.lang.IllegalArgumentException: Type of ValueExpressionPrimary not supported for evaluation!)!
View "dbuser"."Q_StringlengthLocation" could not be archived (  java.sql.SQLException: MS Access query Q_StringlengthLocation is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/211 at [@30,211:218='Language',<168>,1:211]: no viable alternative at input 'Languages.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."Q_StringlengthPatent" could not be archived (  java.sql.SQLException: MS Access query Q_StringlengthPatent is too complex:   java.lang.IllegalArgumentException: Type of ValueExpressionPrimary not supported for evaluation!)!
View "dbuser"."Q_StringlengthPhoto" could not be archived (  java.sql.SQLException: MS Access query Q_StringlengthPhoto is too complex:   java.lang.IllegalArgumentException: Type of ValueExpressionPrimary not supported for evaluation!)!
View "dbuser"."Q_StringlengthSeries" could not be archived (  java.sql.SQLException: MS Access query Q_StringlengthSeries is too complex:   java.lang.IllegalArgumentException: Type of ValueExpressionPrimary not supported for evaluation!)!
View "dbuser"."Q_looseGrouping" could not be archived (  java.sql.SQLException: MS Access query Q_looseGrouping is too complex:   java.lang.IllegalArgumentException: Column type for Q_LOOSEORPHANS.DMSSIGNATURE could not be established!)!
View "dbuser"."Query3 Without Matching QA_ParentChildDetails" could not be archived (  java.sql.SQLException: MS Access query Query3 Without Matching QA_ParentChildDetails is too complex:   java.lang.IllegalArgumentException: Column type for QUERY_SAMMLUNGEN.TECHARCHIVEID could not be established!)!
View "dbuser"."Query4" could not be archived (  java.sql.SQLException: MS Access query Query4 is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/37 at [@7,37:40='Type',<312>,1:37]: no viable alternative at input 'MSysObjects.Type'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."Query_Branches" could not be archived (  java.sql.SQLException: MS Access query Query_Branches is too complex:   java.lang.IllegalArgumentException: Column type for QL_LISTEXTERNALNODES.SIGNATURE could not be established!)!
View "dbuser"."Query_DMSSeriesDistribution" could not be archived (  java.sql.SQLException: MS Access query Query_DMSSeriesDistribution is too complex:   java.lang.IllegalArgumentException: Column type for VIEW_ITS_ARCHIVSCHRÄNKE.DMSID could not be established!)!
View "dbuser"."Query_PhotoModel" could not be archived (  java.sql.SQLException: MS Access query Query_PhotoModel is too complex:   java.lang.IllegalArgumentException: Cannot evaluate set function COUNT(MACHINETYPE.MODEL)!)!
View "dbuser"."Query_SeriesDistribution" could not be archived (  java.sql.SQLException: MS Access query Query_SeriesDistribution is too complex:   java.lang.IllegalArgumentException: Cannot evaluate set function COUNT(TECHNICALARCHIVEOBJECT.TECHARCHIVEID)!)!
View "dbuser"."Query_bibliothek" could not be archived (  java.sql.SQLException: MS Access query Query_bibliothek is too complex:   java.lang.IllegalArgumentException: Column type for "ReportView_ITS_Bibliothek".LIBRARYOBJECTID could not be established!)!
View "dbuser"."Query_signatures" could not be archived (  java.sql.SQLException: MS Access query Query_signatures is too complex:   java.lang.IllegalArgumentException: Column type for QL_LISTINTERNALNODES.SIGNATURE could not be established!)!
View "dbuser"."ReportView_ITS_Archivschränke" could not be archived (  java.sql.SQLException: MS Access query ReportView_ITS_Archivschränke is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/526 at [@59,526:533='Language',<168>,1:526]: no viable alternative at input 'View_ITS_Archivschränke.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."ReportView_ITS_Archivschränke_BAK" could not be archived (  java.sql.SQLException: MS Access query ReportView_ITS_Archivschränke_BAK is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/526 at [@59,526:533='Language',<168>,1:526]: no viable alternative at input 'View_ITS_Archivschränke.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."ReportView_ITS_Bibliothek" could not be archived (  java.sql.SQLException: MS Access query ReportView_ITS_Bibliothek is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/244 at [@31,244:251='Language',<168>,1:244]: no viable alternative at input 'View_ITS_Bibliothek.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."ReportView_ITS_Register" could not be archived (  java.sql.SQLException: MS Access query ReportView_ITS_Register is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/436 at [@59,436:443='Language',<168>,1:436]: no viable alternative at input 'View_ITS_Register.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."ReportView_ITS_Tresor" could not be archived (  java.sql.SQLException: MS Access query ReportView_ITS_Tresor is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/406 at [@59,406:413='Language',<168>,1:406]: no viable alternative at input 'View_ITS_Tresor.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."ReportView_ITS_Zeichnungen" could not be archived (  java.sql.SQLException: MS Access query ReportView_ITS_Zeichnungen is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/481 at [@59,481:488='Language',<168>,1:481]: no viable alternative at input 'View_ITS_Zeichnungen.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."SYS_QueryCategories" could not be archived (  java.sql.SQLException: MS Access query SYS_QueryCategories is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/37 at [@7,37:40='Type',<312>,1:37]: no viable alternative at input 'MSysObjects.Type'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."SYS_QueryList" could not be archived (  java.sql.SQLException: MS Access query SYS_QueryList is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/37 at [@7,37:40='Type',<312>,1:37]: no viable alternative at input 'MSysObjects.Type'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."SYS_QueryList Without Matching SYS_QueryCategories" could not be archived (  java.sql.SQLException: MS Access query SYS_QueryList Without Matching SYS_QueryCategories is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/41 at [@7,41:44='Type',<312>,1:41]: no viable alternative at input 'SYS_QueryList.Type'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."View_ITS_Archivschränke" could not be archived (  java.sql.SQLException: MS Access query View_ITS_Archivschränke is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/605 at [@79,605:612='Language',<168>,1:605]: no viable alternative at input 'Languages.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."View_ITS_Bibliothek" could not be archived (  java.sql.SQLException: MS Access query View_ITS_Bibliothek is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/194 at [@31,194:201='Language',<168>,1:194]: no viable alternative at input 'Languages.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."View_ITS_Register" could not be archived (  java.sql.SQLException: MS Access query View_ITS_Register is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/605 at [@79,605:612='Language',<168>,1:605]: no viable alternative at input 'Languages.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."View_ITS_Tresor" could not be archived (  java.sql.SQLException: MS Access query View_ITS_Tresor is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/605 at [@79,605:612='Language',<168>,1:605]: no viable alternative at input 'Languages.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."View_ITS_Zeichnungen" could not be archived (  java.sql.SQLException: MS Access query View_ITS_Zeichnungen is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/605 at [@79,605:612='Language',<168>,1:605]: no viable alternative at input 'Languages.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
View "dbuser"."View_ITS_build" could not be archived (  java.sql.SQLException: MS Access query View_ITS_build is too complex:   java.lang.IllegalArgumentException: SQL parsing error at position 1/135 at [@19,135:142='Language',<168>,1:135]: no viable alternative at input 'Languages.Language'
Rule stack: [sqlStatement, querySpecification, selectList, selectSublist])!
```

**Comment by HartwigThomas** [comment]
The messages all concern "VIEWS" (Queries, Abfragen).

Unfortunately, Microsoft Access has an extensive Query-Language which goes very far beyond standard SQL, is proprietary and not well documented, and cannot always be handled by SIARD.
We have compromised by displaying errors on the Err tab, when we cannot convert a VIEW, but nevertheless proceed to download the actual data tables.

You should be able to see the content of all tables, but all of the VIEWS mentioned in the error messages are missing.

If the Queries are to be archived, you will have to archive the code separately. We doubt, that the code can be preserved meaningfully for many decades.
If you are mainly interested in archiving the tabular data, the resulting SIARD file should be fine.

I shall test with your database, whether I can see any other problems.

**Comment by HartwigThomas** [comment]
I have tested with your database. The download works correctly. The actual table data are archived. The queries. however, are not all archived. This corresponds to the goal of SIARD to garantee long-term preservation of DATA and not of CODE.
(We do believe that the data will be preserved in a way that is meaningful in 50 years. The MS code, however, will most probably not be executable oder understandable then.

If you want to preserve the text of the queries separately (not as part of an SQL database), you can export them as text and archive them together with the SIARD file.

See e.g. https://stackoverflow.com/questions/1275502/export-all-ms-access-sql-queries-to-text-files

(By the way: I am working in an office which is located in the area of the former Webmaschinen-Fabrik Rüti ...)

**Comment by CarlosHartmann** [comment]
I saw that about your office and found it quite funny. Sending the data back to its former location for further processing etc

I do agree that it seems properly archived in SIARD but a few error messages still confound me.

- After finishing the download I get the message that the file "is invalid".
- KOST-Val returns "Die SIARD-Datei konnte nicht fehlerfrei geoeffnet werden: error in opening zip file. -> Validierung abgebrochen!"
- I thought SIARD is a zipped directory so I tried 7zip which returns "Unexpected end of data", lists every record of table 43, which are indeed most 1KB or empty, and it lists metadata.xml, which is also empty.

That is why I think something is still not quite right. It could have something to do with my installation of the SIARD GUI if you did not encounter any unusual behavior.

**Comment by HartwigThomas** [comment]
I had no problems opening the SIARD file using SIARD.
I cannot really be responsible for KOST-Val. But it would certainly be interesting to know, why it fails.
I also opened it using PKZIP without any problems.

**Comment by HartwigThomas** [comment]
I first tried to open the SIARD file using an old 7ZIP version (2010). I failed to open it. Full 64-bit support came relatively late for many ZIP applications. So I tried with a current version (2021). With that I could open the SIARD file without any problems.

I suspect that the KOST-Val validation fails, because JHOVE have not updated their version verification for SIARD. I tried to convince the people in the Swiss Federal Archives ti sort that out. Unfortunately I probably did not (yet) succeed.

To summarize: I believe, everything is quite right. The SIARD file from you MS Access database can be opened and examined using SIARD, PKZIP and 7ZIP. In all three cases the metadata.xml is complete and readable.
